Dockerfile

```diff
@@ -50,7 +50,9 @@ RUN PYTHONNODEBUGRANGES=1 python -OO -m compileall -b ./python/lib/$runtime/site
 # remove all .py files except ddtrace/contrib/*/__init__.py which are necessary
 # for ddtrace.patch to discover instrumationation packages.
 RUN find ./python/lib/$runtime/site-packages -name \*.py | grep -v ddtrace/contrib | xargs rm -rf
-RUN find ./python/lib/$runtime/site-packages/ddtrace/contrib -name \*.py | grep -v __init__ | xargs rm -rf
+RUN find ./python/lib/$runtime/site-packages/ddtrace/contrib -name "*.py" \
+    -not -path "./python/lib/$runtime/site-packages/ddtrace/contrib/internal/logging/*" \
+    | grep -v __init__ | xargs rm -rf
```

Comment:
Unfortunately, this isn't the right fix. To better understand the problem, take a look at this line change in this PR: https://github.com/DataDog/dd-trace-py/pull/12153/files#diff-6a70144455f9fea41df44b3e578cfaa81ae9c5297e41cce9ef02a5b719013817R265